server/routes/dischargeAddress.js

```diff
@@ -16,16 +17,16 @@ module.exports = function({logger, dischargeAddressService, licenceService, auth
         logger.debug('GET /dischargeAddress');
 
         const nomisId = req.params.nomisId;
-        const licence = await licenceService.getLicence(nomisId);
+        const rawLicence = await licenceService.getLicence(nomisId);
+        const licence = getIn(rawLicence, ['licence']);
 
-        if(licence.length < 1) {
+        if(!licence) {
```

Comment:
now this one is a controversial one so feel free to ignore it if you want... but i'm not a big fan of the `!` it's too easy to get missed.. now for the controversial part.. ramda as a utility function called `not` which is reminisent to pyhton or haskell... you could also easly write your own... 

`if(!licence) {`

would become

`if(not(licence)) {`

Again ignore this if you want... just been a JS pendantic about it ahah

Reply:
I have heard this opinion before. Not sure that I agree though! 

Surely it is a part of the language that has been around forever and having to import something to do the same thing is over the top. Also missing it would completely change the functionality of the function and would be caught by any test